### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (.ui-datepicker-trigger:nth-child(4))

### ISSUE THREAD

**Issue by AdnoC** [AdnoC/Dotfiles#5]
**Issue**: `Buttons must have discernible text` ([`button-name`](https://dequeuniversity.com/rules/axe/3.1/button-name?application=axeAPI))

**Target application**: [Mars Commuter: Travel to Mars for Work or Pleasure!](https://dequeuniversity.com/demo/mars/)

**Element path**: .departure-date > .ui-datepicker-trigger:nth-child(4)

**Snippet**:

    <button class="ui-datepicker-trigger" type="button">  </button>

**How to fix**:

    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element has a value attribute and the value attribute is empty
      Element has no value attribute or the value attribute is empty
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
      Element has no title attribute or the title attribute is empty

**Environment**:
Chrome version 71.0.3578.98

====

This accessibility issue was found using Accessibility Insights for Web, a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
